packages/ui/client/components/views/ViewConsoleOutputEntry.cy.tsx

```diff
@@ -0,0 +1,42 @@
+import Filter from 'ansi-to-html'
+import ViewConsoleOutputEntry from './ViewConsoleOutputEntry.vue'
+
+const htmlSelector = '[data-type=html]'
+const textSelector = '[data-type=text]'
+
+describe('ViewConsoleOutputEntry', () => {
+  it('test plain entry', () => {
+    const content = new Date().toISOString()
+    const container = cy.mount(
+      <ViewConsoleOutputEntry
+        task-name="test/text"
+        type="stderr"
+        time={Date.now()}
+        html={false}
+        content={content}
+      />,
+    ).get(textSelector)
+    container.should('exist')
+    container.invoke('text').then((t) => {
+      expect(t, 'the message has the correct message').equals(content)
+    })
+  })
```

Comment:
The goal this test is to make sure that there is text that has the `content` you pass in when HTML is false. You also want to make sure that the `type`, `task-name`, and `time` are visible. Here's the shortest way to write that test:

```suggestion
  it('test plain entry', () => {
    const content = faker.hacker.phrase()
    const time = Date.now()
    const localeTime = time.toLocaleString()
    const type = 'stderr'
    const taskName = 'test/text'
    cy.mount(
      <ViewConsoleOutputEntry
        task-name={taskName}
        type={type}
        time={time}
        html={false}
        content={content}
      />,
    )

    // always either check for visible or exists
    // visibility checks prevent CSS bugs relating to overflow, height, width, or opacity
    cy.contains(content).should('be.visible')
    cy.contains(taskName).should('be.visible')
    cy.contains(localeTime).should('be.visible')
    cy.contains(type).should('be.visible')
  })
  
```